example/README.md

```diff
@@ -1,8 +1,44 @@
 # intercom_example
 
-Demonstrates how to use the intercom plugin.
+Demonstrates how to use the Intercom plugin in a Flutter project. 
 
-## Getting Started
+## Initialize Intercom & Identify User
+
+See example code at [lib/main.dart](lib/main.dart)
+
+configure your app's main function as follows
+
+```dart
+void main() async {
+    await Intercom.initialize('appId',
+      androidApiKey: 'androidApiKey', iosApiKey: 'iosApiKey');
+
+    await Intercom.registerIdentifiedUser(email: 'user_email@example.com'); 
+
+    // you can also use the userId instead of email address 
+    // await Intercom.registerIdentifiedUser(userId: 'intercomUserID'); 
+
+    runApp(MyApp());
+  }
+```
+
+## Display Intercom Messenger from Flutter UI Element
+
+```dart
+new FlatButton(
```

Comment:
`new` is no longer required in Dart

Reply:
Yes I know this, was referencing the example code in your repo.